Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSIdentityPropertyKeys.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Smithy.AttributeKey
+
+public enum AWSIdentityPropertyKeys {
```

Comment:
These keys are used to pass data & components into identity resolvers in a type-safe manner.